### PR TITLE
Fixed style to self guided tour to fit the Designs

### PR DESCRIPTION
**Pull request by alefaissal**
# Fixed style to self guided tour to fit the Designs

### CODE REVIEW

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js [hunk 1]
```diff
@@ -24,54 +24,64 @@ const SelfGuidedItem = ({selfGuidedItem, navigation}) => (
...
+                        if (
+                          savedIds.some(
+                            savedId => savedId === selfGuidedItem.id,
+                          )
+                        ) {
+                          removeSaveId(selfGuidedItem.id);
+                        } else {
+                          addSaveId(selfGuidedItem.id);
+                        }
+                      } catch (error) {}
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/SelfGuidedItem.js]: What should happen when there is an error? 
Are there any promises that you need to await? Please clarify.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the requested changes.

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js (hunk 1 above)
Comment (alefaissal) [reply client/js/components/SelfGuidedItem/SelfGuidedItem.js]: Add a Throw new Error there.

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js [hunk 2]
```diff
@@ -24,54 +24,66 @@ const SelfGuidedItem = ({selfGuidedItem, navigation}) => (
...
-                    }
+                    style={styles.contactSocialMediaIconShare}
+                    source={require('../../assets/images/icFaveShareDefault.png')}
                   />
-                </TouchableOpacity>
+                  <TouchableOpacity
+                    onPress={async e => {
+                      try {
+                        if (
+                          await savedIds.some(
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/SelfGuidedItem.js]: saveIds is an array. I don't think you need await on any of these methods. Remove asyn await try catch.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js (hunk 2 above)
Comment (alefaissal) [reply client/js/components/SelfGuidedItem/SelfGuidedItem.js]: Removed

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**